apps/agent/README.md

```diff
@@ -0,0 +1,128 @@
+# Agent
+
+
+
+## 1. Mechanism
+
```

Comment:
It may be interesting to include a section on "security features" when describing the app, as the Agent specifically comes with some (`safeExecute()`).